pypeit/scripts/ql_multislit.py

```diff
@@ -5,12 +5,11 @@
 .. include:: ../include/links.rst
 """
 
-import os
 import copy
+import os
```

Comment:
Is `os` still needed?

Reply:
Not anymore... changed line `print_offset_report()` that was still using it.

Reply:
Actually, yes, `os` is still needed.